assets/dev/scss/editor/_global.scss

```diff
@@ -166,3 +165,31 @@
 i.eicon-tilted {
 	transform: rotate(90deg);
 }
+
+.elementor-control .elementor-control-content.elementor-control-media-size {
+	margin-top: 15px;
+}
+
+.elementor-control-media-size .elementor-control-field {
+	padding: 0px;
+}
+
+.elementor-control-media-size .elementor-control-responsive-switchers {
+	display: none;
+}
+
+.elementor-control-media-display-size.select {
+	padding-bottom: 0px;
+	width: 72%;
+	float: right;
+}
+
+.media-size-input-wrapper {
+	position: unset;
+    float: right;
+    width: 90%;
+}
+
+.elementor-control-media-size .elementor-control-title {
+	width: 32%;
```

Comment:
same as above regarding the magic number

Reply:
The panel has x Width, the select need to be 32% of his wrapper width in order to be aligned with all the other inputs.